content/content-strategy/creating-user-centred-content/using-existing-research/existing-research.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: text/textblock
+section: Why use existing research?
+---
+When creating content, consider all the possible users of your product or service. Existing research will help you find out about your users and their needs. This existing research can complement your own research. It is important to have a good understanding of both to create content that is user friendly and of value.
+
+Using existing research has many benefits:
+- it can save time and money 
+- it can provide you with evidence of who your users are and their needs in the initial phase of a project
+
+This can help gain support in your organisation for:
+- following a user-centred design process
+- allocating budget and resources to a project
+
+Existing research is a good starting point, but it shouldn’t be relied on as comprehensive.  Sometimes sources may be limited or dated. Doing your own user research is the best way to learn about users and create services that meet their needs.
+
+See also [Getting buy-in]()
```

Comment:
There's a missing hyperlink on Line 17 for 'Getting buy-in' text. The page exists, so best to hook it up.